### DISCUSSION TITLE
about aria2 in scoop

### DISCUSSION THREAD

**Discussion by ghost** [ScoopInstaller/Scoop#5756] [Help and Questions] [answered]
[scoop use aria2 ](https://github.com/ScoopInstaller/Scoop#multi-connection-downloads-with-aria2)
can I create a aria config file?
Instead of using `scoop config`
eg :
`scoop config aria2-max-connection-per-server 16`
If it is possible to support the creation of personal profiles. where should it be placed?

**Comment by HUMORCE** [comment, marked as answer]
> can I create a aria config file?

No, but we can set any options with `scoop config aria2-options @("--option-1=true", "--option-1='00 11 22'", ...)`

https://github.com/ScoopInstaller/Scoop/blob/f93028001fbe5c78cc41f59e3814d2ac8e595724/lib/install.ps1#L182-L234

**Reply by ghost** [reply]
okay. thanks.
That is to say, even if I create a configuration file, scoop will not use it, right?

**Reply by HUMORCE** [reply]
> even if I create a configuration file, scoop will not use it, right?

It may be in effect? I don't know, I don't enable aria2 for Scoop. You can try specifying a invalid proxy server in aria2's default config file then execute `scoop download aria2` to verify this.

**Reply by hrxn** [reply]
Unless the scoop project made any modifications to its distributed aria2 - which is very unlikely, I think - then yes, aria2 *will* use it's normal configuration file paths.
